templates/basic/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "type": "module",
+  "scripts": {
+    "dev": "react-router dev",
+    "build": "react-router build",
+    "start": "react-router-serve ./build/server/index.js",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@react-router/node": "0.0.0-nightly-8f12ed19a-20240924",
+    "@react-router/serve": "0.0.0-nightly-8f12ed19a-20240924",
+    "@tailwindcss/vite": "^4.0.0-alpha.24",
+    "isbot": "^5.1.17",
```

Comment:
I removed a bunch of dependencies (like eslint). I don't know if `isbot` is a peer dep or what, but that's the only one I kept I wasn't sure about

Also moved to tailwind alpha since it removes a bunch of files and I imagine it'll be released before we release v7 official. If that's no the case that's fine, I'll just switch back

Reply:
isbot is used on the entry.server on Remix apps, I don't see that file here, is it removed? Or just hidden?

Reply:
> like eslint

Thank you. 🙏

Reply:
> isbot is used on the entry.server on Remix apps, I don't see that file here, is it removed? Or just hidden?

Yeah, I removed `entry.server` and `entry.client`. This is how Remix worked, if they're not there a hidden version is automatically used.

We may add them back as default visible, I was just trying to make this super simple